src/main/java/duckling/ResponseHeaders.java

```diff
@@ -0,0 +1,37 @@
+package duckling;
+
+public class ResponseHeaders {
+    protected String contentType = "null";
+    protected String status = "200 OK";
+
+    public String[] toArray() {
+        return new String[] {
+            "HTTP/1.0 " + status + Server.CLRF,
+            "Content-Type: " + contentType + Server.CLRF,
+            Server.CLRF
+        };
+    }
+
+    public ResponseHeaders notFound() {
+        return withStatus("404 NOT FOUND");
+    }
+
+    public ResponseHeaders withContentType(String givenContentType) {
+        String currentStatus = status;
+
+        return new ResponseHeaders() {{
+            status = currentStatus;
+            contentType = givenContentType;
+        }};
```

Comment:
That's reasonable - there's not a _whole lot_ to modify in response headers as well.  I've been sort of trained to steer towards keyword arguments by years of Ruby/JS, so I guess this was my habits trying to exert themselves.